lib/genevalidator/output.rb

```diff
@@ -224,6 +227,25 @@ def calculate_overview(overview)
         [eval, error_eval, time_eval].reject(&:empty?)
       end
 
+      def write_summary_file(overview, summary_file, opt)
+        return unless opt[:output_formats].include? 'summary'
+        data = generate_summary_data(overview)
+        File.open(summary_file, 'w') { |f| f.write data.map(&:to_csv).join }
+      end
+
+      def generate_summary_data(overview)
+        [
+          ['total.queries', overview[:no_queries]],
+          ['good.scores', overview[:good_scores]],
+          ['bad.scores', overview[:bad_scores]],
+          ['no.of.queries.with.more.than.20.BLAST.hits',
```

Comment:
- What's the relevance of number of queries with more than 20 hits?
- I thought we wanted a 'number of predictions with no evidence'
- Can we have underscores instead of periods in the names?
- How about naming the keys: 'num_predictions', 'num_good_predictions', 'num_bad_predictions', 'num_predictions_with_insufficient_evidence', 'first_quartile_of_scores', 'second_quartile_of_scores', ...